tests/test_definitions/column_map_expectations/expect_column_values_to_be_of_type.json

```diff
@@ -33,6 +33,12 @@
         },
         "redshift": {
           "x": "INTEGER"
+        },
+        "bigquery": {
+          "x": "INT64"
+        },
+        "databricks": {
+          "x": "INT"
```

Comment:
@TrangPham could we confirm that these new schemas are actually running? I see a `only_for: ["pandas"]` below on (line 56) and suspect it might be skipping most of them.